LibraBFT/Prelude.agda

```diff
@@ -46,6 +46,8 @@ module LibraBFT.Prelude where
     hiding (fromMaybe; [_])
     public
 
+  fmap = List-map
```

Comment:
I think this should be an alias for `_<$>_`

Reply:
Agreed.  I will do that.